pkg/resource/alias/hooks.go

```diff
@@ -24,6 +24,8 @@ import (
 	svcapitypes "github.com/aws-controllers-k8s/lambda-controller/apis/v1alpha1"
 )
 
+// syncFunctionEventInvokeConfig calls `PutFunctionEventInvokeConfig` to update the fields
```

Comment:
```suggestion
// syncEventInvokeConfig calls `PutFunctionEventInvokeConfig` to update the fields
```
in go function comments have to always start with function "exact" name. https://tip.golang.org/doc/comment

Reply:
Thanks!